sync/src/tasks/block_sync_task.rs

```diff
@@ -513,13 +501,8 @@ where
                             process_dag_ancestors
                                 .insert(block.header().id(), block.header().clone());
 
-                            // execute the children if their parents are ready
-                            // if all children are executed, delete it.
                             self.execute_if_parent_ready(executed_block.block.id())?;
 
-                            // todo: how to delete the block?
-                            // self.local_store.delete_dag_sync_block(executed_block.block.id())?;
```

Comment:
It seems that you don't think clearly how to process usefuless dag_sync_block.You can't remove this comment.